packages/shared/src/variants/baduk.ts

```diff
@@ -244,6 +273,10 @@ export class Baduk extends AbstractGame<NewBadukConfig, BadukState> {
     return this.sgf?.sgfContent ?? "non-rectangular";
   }
 
+  /**
+   * Initialises a Ko detector. Override this to change the Ko detection method.
```

Comment:
```suggestion
   * Override this to change the Ko detection method.
```

This is another case where I feel the name of the method pretty much said it already.

On the other hand, the notes about why one would override it are valuable.